source/sonata_config.rst

```diff
@@ -226,6 +228,7 @@ example::
                    "type": "biophysical",
                    "morphologies_dir": "...",
                    "biophysical_neuron_models_dir": "...",
+                   "spatial_index_dir": "...",
```

Comment:
Unrelated to this MR, can you please fix also the nested level of "nodes"? It should be under "networks".